sources/pmwebapi.go

```diff
@@ -137,6 +137,17 @@ func newPcpSource() (PcpSource, error) {
 			log.Fatal("NewRequest: ", err)
 		}
 
+		unit := ""
+		if pcpMetrics.Units != "" {
+			unit = "_" + pcpMetrics.Units
+		}
+		if strings.ToUpper(pcpMetrics.Type) == "COUNTER" {
+			pcpMetrics.Name = pcpMetrics.Name + unit + "_total"
```

Comment:
If you want to make this even cleaner, you can use the `+=` operator. This adds the variable you are storing the result to at the beginning of the operation. So...
```go
pcpMetrics.Name += unit + "_total"
```
is shorthand for
```go
pcpMetrics.Name = pcpMetrics.Name + unit + "_total"
```